kotest-core/src/jvmMain/kotlin/io/kotest/core/engine/SpecRunner.kt

```diff
@@ -7,10 +7,10 @@ import io.kotest.core.test.TestCase
 import io.kotest.core.test.TestResult
 import io.kotest.fp.Try
 import io.kotest.mpp.log
-import kotlinx.coroutines.runBlocking
-import java.util.concurrent.ExecutionException
-import java.util.concurrent.Executors
-import java.util.concurrent.TimeUnit
+import java.util.concurrent.*
```

Comment:
Minor: Your indentations and `*` import way show that your IDE configuration is different from the others. Maybe you can validate this with other developers of this project.

Reply:
Thanks for catching this!
You're correct, I had an incorrect configuration compared to normally after a fresh install on a new machine.